### PR TITLE
Add error-patterns for ATS warnings

### PR DESCRIPTION
**Pull request by marsam**
Hi:
I've added support warning messages

### CODE REVIEW

File: flycheck-ats2.el [hunk 1]
```diff
@@ -51,41 +51,48 @@
...
+    space ,level "(" (id (one-or-more alnum)) "):"
     ;; error message; up to three lines long
     (message
      (+? not-newline)
      (or (: ?: ?\n (repeat 2 (: (1+ not-newline) ?\n)))
          ?\n))))
-  :modes ats-mode)
+
+(flycheck-define-command-checker 'ats2
+  "Flycheck checker for F*."
```
Comment (drvink) [inline flycheck-ats2.el]: @marsam I think you meant ATS2 here. :)

**Comment by drvink** [comment]
Thanks for the contribution. If you can fix the one comment I made on the commit, I'll gladly merge this.

PS. Is that Houtarou in your avatar? :)

**Comment by marsam** [comment]
Fixed
> PS. Is that Houtarou in your avatar? :)

yes :)

**Comment by drvink** [comment]
Thought so. I love Hyouka. :)

Thanks for the commit!

**Pull request merged**